### DISCUSSION TITLE
authorization in home connect

### DISCUSSION THREAD

**Discussion by Taronaga** [ekutner/home-connect-hass#199] [Community Support]
Hi all. AFTER entering the e-mail, I get the following error

![image](https://user-images.githubusercontent.com/62870792/228820594-7702a155-e24e-4d3b-8ee8-86472fb884e5.png)

**Comment by nordeep** [comment]
Check next Issue - https://github.com/home-assistant/core/issues/81481#issuecomment-1500098028
Seems it's like miss some hidden input values. My investigation showed that only the `password` value is missing.
